apps/native/app/jest.config.ts

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  displayName: 'IslandApp',
```

Comment:
~Should match nx project name?

Reply:
This is the same as is declared in app.json (and was like that before) so I am keeping as is.